README.md

```diff
@@ -22,14 +22,33 @@ Get help inforamtion for available commands,
 Initialize the dependencies by coping the holochain binaries (**Required**),
 
 ```shell
+# Initializes the ad4m-host with default config
 ./ad4m init
+
+# Initalize ad4m-host with different environment
+./ad4m init --dataPath environment_name --networkBootstrapSeed ./custom_seed.json
```

Comment:
does the `environmen_name` mean `ad4m`?
we better give a concrete and working example here I think.